framework/src/modules/chain/helpers/z_schema.js

```diff
@@ -393,6 +392,23 @@ const liskFormats = {
 			);
 		},
 	},
+	/**
+	 * Returns true if value is lower than or equal to `TOTAL_AMOUNT`.
+	 *
+	 * @param {Object} value
+	 * @returns {boolean}
+	 */
+	maxAmount: {
```

Comment:
`maxAmount` would be `TOTAL_AMOUNT`. The validation is checking if it's a correct amount - positive and lower than `TOTAL_AMOUNT`. 

I suggest to rename it to `amount`.

Reply:
Not possible to define both `string` and `number` for a custom schema.
I changed it to `numAmount` therefore as `amount` is already been taken by the amount validator for strings.